exist-parent/pom.xml

```diff
@@ -104,7 +104,7 @@
         <apache.httpcomponents.version>4.5.13</apache.httpcomponents.version>
         <apache.httpcomponents.core.version>4.4.14</apache.httpcomponents.core.version>
         <apache.xmlrpc.version>3.1.3</apache.xmlrpc.version>
-        <aspectj.version>1.9.4</aspectj.version>
+        <aspectj.version>1.9.8.M1</aspectj.version>
```

Comment:
@reinhapa I guess you saw this one - https://bugs.eclipse.org/bugs/show_bug.cgi?id=553623  ?

Reply:
Wow, two years. @reinhapa might have picked up my workaround from
https://github.com/eXist-db/exist/pull/4084/commits/fe77cd696da550e3709fa01136eb415bc9b21360